app/src/main/java/openfoodfacts/github/scrachx/openfood/views/HistoryScanActivity.java

```diff
@@ -155,9 +162,33 @@ public void exportCSV() {
             }
             writer.close();
             Toast.makeText(this, R.string.txt_history_exported, Toast.LENGTH_LONG).show();
+            isDownload = true;
         } catch (IOException e) {
             e.printStackTrace();
         }
+
+        NotificationManager notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        Intent downloadIntent = new Intent(Intent.ACTION_VIEW);
+        downloadIntent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_NEW_TASK);
+        Uri csvUri = FileProvider.getUriForFile(this, this.getPackageName() + ".provider", f);
+        downloadIntent.setDataAndType(csvUri, "text/csv");
+        downloadIntent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
+
+        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.O) {
+            int importance = NotificationManager.IMPORTANCE_DEFAULT;
+            NotificationChannel notificationChannel = new NotificationChannel("downloadChannel", "ChannelCSV", importance);
+            notificationManager.createNotificationChannel(notificationChannel);
+        }
+
+        NotificationCompat.Builder builder = new NotificationCompat.Builder(this, "downloadChannel")
+                .setContentTitle("Download Complete")
+                .setContentText("Open the File to view the exported products")
```

Comment:
Both of these strings need to be translatable.

Maybe we should have the message as
> Export complete
> Click to view exported products

Reply:
agreed. Download implies we store your scan history on the server, which is not true.

Reply:
@Karljoones @teolemon  I will make the changes.